jaq-std/src/std.jq

```diff
@@ -22,6 +22,20 @@ def isinfinite: . == infinite or  . == -infinite;
 def isfinite:   isnumber and (isinfinite | not);
 def isnormal:   isnumber and ((. == 0 or isnan or isinfinite) | not);
 
+# Math
+def logb:
+    if . == 0.0 then -infinite
+  elif isinfinite then infinite
+  elif isnan then .
+  else ilogb | . + 0.0 end;
+def pow10:       pow(10.0; .);
+def scalb(x; e): x * pow(2.0; e);
+def significand:
+    if isinfinite or isnan then .
+  elif . == 0.0 then 0.0
```

Comment:
You could merge this with the line above.

Reply:
I wrote it like this to ensure the output type is always float :thinking:. It otherwise yields floats except when given `0` as an integer.

Reply:
Ah, I see. Thanks for clarifying this.